src/gw_gpu_kernel.cpp

```diff
@@ -119,6 +122,39 @@ namespace green::gpu {
       MPI_Op_free(&matrix_sum_op);
     }
 
+    void gw_gpu_kernel::flops_achieved(MPI_Comm comm) {
+      double gpu_time=0.;
+      if (comm != MPI_COMM_NULL) {
+        utils::event_t& cugw_event = statistics.event("Solve cuGW");
+        if (!cugw_event.active) {
+          gpu_time = cugw_event.duration;
+        } else {
+          gpu_time = 0.;
+          cugw_event.active = false;
+          throw std::runtime_error("'Solve cuGW' event not found");
```

Comment:
Kinda misleading error message, you get here if `cugw_event.active` is true, so technically `Solve cuGW` event does exist, but it is still actively collecting data.

Reply:
Agreed. Made changes to the error message. But in general, should we even keep the `if (!cugw_event.active)` condition? I initially put the condition and error message so that I do not end up introducing any bugs.